roles/kolibri/tasks/install.yml

```diff
@@ -4,12 +4,31 @@
 # https://github.com/learningequality/pi-gen/blob/master/stage2/04-hostapd/offline.yml
 # https://github.com/learningequality/pi-gen/blob/master/stage2/04-hostapd/online.yml
 
+# Install Kolibri » Debian/Ubuntu
+# https://kolibri.readthedocs.io/en/latest/install/ubuntu-debian.html
+
+# Advanced management
+# https://kolibri.readthedocs.io/en/latest/manage/advanced.html
+
+# Working with Kolibri from the command line
+# https://kolibri.readthedocs.io/en/latest/manage/command_line.html
+
+# Customize Kolibri settings with the [ /library/kolibri/options.ini ] file
+# https://kolibri.readthedocs.io/en/latest/manage/options_ini.html
+
+# Test Kolibri server performance
+# https://kolibri.readthedocs.io/en/latest/manage/performance.html
+
+# Provisioning many servers
+# https://kolibri.readthedocs.io/en/latest/install/provision.html
+
+
 - name: Create Linux user {{ kolibri_user }} and add it to groups {{ apache_user }}, disk
   user:
     name: "{{ kolibri_user }}"
     groups:
       - "{{ apache_user }}"
-      - disk
```

Comment:
@holta Security feature of debian. Please refer https://wiki.debian.org/SystemGroups

Reply:
If group disk is needed for USB drive access, yes we should annotate that in future.

This root-style access level might or might not be fully necessary (requirement seems to remains somewhat ambiguous?)

As noted @ 190ac34

Reply:
Just FYI/FWIW: if one allows `apt install kolibri` to create and set up a new KOLIBRI_USER, it does not add the user to _any_ Linux groups (like `www_data` or `disk`).

[*] i.e. if installing a Kolibri .deb package at the command-line (outside of Ansible) and without setting `export DEBIAN_FRONTEND=noninteractive` — so as to be presented with up-to-5 debconf interactive screens.

Reply:
Another FYI/FWIW: Kolibri _appears to work just fine_ (with IIAB) when KOLIBRI_USER  is not added to _any_ Linux groups (like `www-data` or group `disk`).

However I only tested on a VM (Ubuntu 22.04).  So further testing would be required to see if Kolibri's USB drive functionality works or not.  Which of course still [requires](https://github.com/iiab/iiab/blob/660ee81e281295b48fa1abbb5160e39d8a1e5544/vars/default_vars.yml#L322-L324):

```
# Kolibri can export & import channels to USB sticks/drive:
usb_lib_umask0000_for_kolibri: True  
```

As explained in [usb_lib/README.rst](https://github.com/iiab/iiab/blob/master/roles/usb_lib/README.rst) here:

> IIAB will generally mount USB drives 'rw' allowing root to both read and write to them.  In addition, in March 2021 (`PR #2715 <https://github.com/iiab/iiab/issues/2715>`_) Kolibri exports were enabled by also giving non-root users read and write access to VFAT/FAT32, NTFS and exFAT USB drives, using ``umask=0000`` (in /etc/usbmount/usbmount.conf) to override the ``umask=0022`` default.  If however you prefer to restore usbmount's default, set ``usb_lib_umask0000_for_kolibri: False`` in `/etc/iiab/local_vars.yml <http://FAQ.IIAB.IO/#What_is_local_vars.yml_and_how_do_I_customize_it.3F>`_ (preferably do this prior to installing IIAB).

Reply:
Quick context for `Group=www-data` on Line 12 of /etc/systemd/system/kolibri.service :

> meaning of systemd "Group" option
> https://serverfault.com/questions/805879/meaning-of-systemd-group-option/805883#805883

Reply:
Linux group `disk` is proven to be unneeded with USB stick testing (using 64-bit Raspberry Pi OS, on a Raspberry Pi 4).

So I will remove it entirely.

(It would appear this root-level access was never needed from day one, and it's time to take security more seriously in 2023 now!)